extensions/script-libraries/extension-utils/extension-utils.livecodescript

```diff
@@ -0,0 +1,1634 @@
+﻿script "com.livecode.library.extension-utils"
+/*
+Copyright (C) 2018 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>. 
+*/
+
+/**
+Title: Extension Package Handling Utilities
+
+Type: library
+
+Summary: This library provides handlers for managing livecode extension packages
+
+Version: 1.0.0
```

Comment:
@livecodeali do we have a flag we can add here to indicate this is an IDE library?

Reply:
Not yet I don't think...